### PR TITLE
Content Review: Upgrade Guide

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8366

### CODE REVIEW

File: modules/upgrade/pages/client-migration.adoc [hunk 1]
```diff
@@ -1,184 +1,153 @@
...
-
-To upgrade the SP version on SLE{nbsp}12 (for example, upgrading from SLE{nbsp}12 or any SLE{nbsp}12{nbsp}SPx to SLE{nbsp}12{nbsp}SP4) can be fully automated and requires no additional preparation.
-
-
-
-[[bp.client.migration.sle]]
-== Upgrading SLE{nbsp}12{nbsp}SPx to Version 15
+[NOTE]
+====
+This process cannot be used to upgrade Salt clients
```
Comment (keichwa) [inline modules/upgrade/pages/client-migration.adoc]:
```suggestion
This process cannot be used to upgrade Salt clients.
```

File: modules/upgrade/pages/client-migration.adoc [hunk 2]
```diff
@@ -1,184 +1,153 @@
...
-.Auto-Upgrading Salt Clients Currently Not Supported
-[IMPORTANT]
-====
-This procedure will work for traditionally  managed systems (system type [systemitem]``management``).
-It is not currently available for systems using Salt (system type [systemitem]``salt``).
-====
 
-During the procedure, the machine reboots and performs the system upgrade.
-The process is controlled by {yast} and {ay}, not by [command]``zypper`` commands.
+// We should probably split the SLE12->SLE12 and SLE12->SLE15 instructions. LKB 2019-08-22
```
Comment (keichwa) [inline modules/upgrade/pages/client-migration.adoc]: Here we only need SLE12->SLE15 (SP0) and a forward-looking remark about SLE15 (SP0) -> SLE15 (SP1) (and later).

Then we need short statements, how a user can migrate older clients (11 or 12) to a supported version (either LTSS or SLE 15).

File: modules/upgrade/pages/client-migration.adoc [hunk 3]
```diff
@@ -1,184 +1,153 @@
...
 . Go to menu:Main Menu[Systems > Activation Keys] and click [guimenu]``Create Key``.
 . Enter a description for your key.
 . Enter a key or leave it blank to generate an automatic key.
-. If you want to limit the usage, enter your value in the [guimenu]``Usage`` text field.
+. OPTIONAL: If you want to limit the usage, enter your value in the [guimenu]``Usage`` text field.
 . Select the [systemitem]``SLE-Product-SLES15-Pool for x86_64`` base channel.
-. Decide about [guimenu]``Add-On System Types``.
-If in doubt, see https://www.suse.com/documentation/sles-15/book_quickstarts/data/art_modules.html ({slea} Modules & Extensions Quick Start).
+. OPTIONAL: Select any [guimenu]``Add-On System Types``.
+For more on this, see https://www.suse.com/documentation/sles-15/book_quickstarts/data/art_modules.html.
```
Comment (keichwa) [inline modules/upgrade/pages/client-migration.adoc]:
```suggestion
For more information, see https://www.suse.com/documentation/sles-15/book_quickstarts/data/art_modules.html.
```
;)

File: modules/upgrade/pages/client-migration.adoc [hunk 4]
```diff
@@ -1,184 +1,153 @@
...
-Check the menu:System Status[] on the system's details when the migration is done.
-
-image::sles_migrated.png[scaledwidth=80%]
+You can check the progress of the migration by navigating to menu:System Details[System Status].
 
-If the menu:System Status[] notification does not report a successful migration but lists menu:Software Updates Available[], install the update now and then check again.
+If the system status reports that the migration has failed, but also shows that software updates are available, install the update and check the status again.
 
-Finally, consider to schedule a reboot.
+After the migration is complete, reboot the client.
```
Comment (keichwa) [inline modules/upgrade/pages/client-migration.adoc]: We must make sure that users will get this right.  The user must not have the impression that he has to do this client-side. He should schedule a reboot in SUMA.

File: modules/upgrade/pages/migrate-3x-3x.adoc [hunk 5]
```diff
@@ -1,98 +1,92 @@
...
-In case of trouble, resolve the following issues first:
+.Procedure: Migrating with {yast}
+
+. From the command prompt, as root, launch the {yast} online migration tool:
++
+----
+yast2 migration
+----
+{yast} will show the possible migration targets with detailed summaries.
+// I've removed the graphical option, because we tell people to use a text console. LKB 2019-08-21
```
Comment (keichwa) [inline modules/upgrade/pages/migrate-3x-3x.adoc]: :+1:

File: modules/upgrade/pages/client-migration.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/upgrade/pages/client-migration.adoc]: Yep, that makes sense. I'll open an issue to track this.

Comment (Loquacity) [reply modules/upgrade/pages/client-migration.adoc]: https://github.com/SUSE/spacewalk/issues/9153

**Comment by Loquacity** [comment]
Another one bites the dust! :rocket:

File: modules/upgrade/pages/client-migration.adoc [hunk 6]
```diff
@@ -1,184 +1,153 @@
...
-Check the menu:System Status[] on the system's details when the migration is done.
-
-image::sles_migrated.png[scaledwidth=80%]
+You can check the progress of the migration by navigating to menu:System Details[System Status].
 
-If the menu:System Status[] notification does not report a successful migration but lists menu:Software Updates Available[], install the update now and then check again.
+If the system status reports that the migration has failed, but also shows that software updates are available, install the update and check the status again.
 
-Finally, consider to schedule a reboot.
+After the migration is complete, schedule a reboot of client from the {productname} Server.
```
Comment (keichwa) [inline modules/upgrade/pages/client-migration.adoc]:
```suggestion
After the migration is complete, schedule a reboot of the client from the {productname} Server.
```

**Pull request merged**